pkg/storage/tsdb/config.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/alecthomas/units"
+	errors2 "github.com/pkg/errors"
```

Comment:
Can we remove the import of `"errors"` in this file so that we don't have to name it `errors2`?